README.md

```diff
@@ -108,6 +109,36 @@ const result = parseMarker('db', `
 `)
 ```
 
+### Metadata parsing
+
+Metadata uses the same syntax as GraphQL directives.
+
+Usage: 
+
+```js
+const result = parseMetadata('db', `
+  This is a description
+  @db(length:200, 
+      unique: true, 
+      columns: ['id', 'name']
+      description: \`
```

Comment:
Actually I wonder do we even need to cater for this use case. We don't want to encourage super complex, verbose metadata.

Reply:
Yes... all of this is really overkill. Content here should be available as son.